app/controllers/salt_controller.rb

```diff
@@ -0,0 +1,29 @@
+require "velum/salt"
+require "velum/salt_api"
+# HealthController holds methods for checking the readiness of the application.
+class SaltController < ApplicationController
+	include Velum::SaltApi
+ 	skip_before_action :redirect_to_setup
+
+	def update
+
+    minions = perform_request(endpoint: "/minions", method: "get")
+    ret = JSON.parse(minions.body)["return"]
+
+    for minion in ret do
```

Comment:
We usually use the `ret.each do |minion|` syntax, but affter using `Salt.minions` you won't need to do this I think.